### PR TITLE
DOCSP-27817 manual tags for shell top 250

### PR DESCRIPTION
**Pull request by sarahemlin**
Hey @sarah-olson-mongodb & @mdb-ashley!

Here are the manual tags for shall pages in the top 250.  Please merge if everything looks good to you!

Thank you!
Sarah

## JIRA
https://jira.mongodb.org/browse/DOCSP-27817

### CODE REVIEW

File: source/crud.txt [hunk 1]
```diff
@@ -6,6 +6,13 @@ Perform CRUD Operations
 
 .. default-domain:: mongodb
 
+   .. facet::
+   :type: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: server
```
Comment (sarah-olson-mongodb) [inline source/crud.txt:14]: Thanks @sarahemlin, do we have a keyword for the Shell?

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @sarahemlin! LGTM with one minor question.

File: source/crud.txt (hunk 1 above)
Comment (sarahemlin) [reply source/crud.txt:14]: The [Official Taxonomy](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy?src=contextnavpagetreemode) has shell as a programmingLanguage value.

**Review by mdb-ashley — APPROVED** [review]
LGTM, thanks Sarah!

**Review by sarahemlin — COMMENTED** [review]
facet::name fix and spacing check on all files

**Pull request merged**